app.js

```diff
@@ -0,0 +1,22 @@
+import mu from 'mu';
+
+const app = mu.app;
+const bodyParser = require('body-parser');
+const repository = require('./repository');
+const cors = require('cors');
+const debug = process.env.DEBUG_LOGGING || false;
+
+app.use(bodyParser.json({ type: 'application/*+json' }));
+app.use(cors());
```

Comment:
We doen tot nog toe geen CORS requests. Deze mag dus ook weg

Reply:
Cleanup gedaan